src/Kernel/Object.class.st

```diff
@@ -1578,6 +1578,18 @@ Object >> okToChange [
 	^ true
 ]
 
+{ #category : #'as yet unclassified' }
```

Comment:
Uncategorized methods, causes tests failures

Reply:
I just recategorised this method, as well as two others that were not categorised. I am not sure why the recategorisation of okToChange does not appear in the commit though. https://github.com/pharo-project/pharo/pull/2032/commits/73630e39810a83f2957dcfc95c24ed77e2a60f43

Reply:
I tried to reload my branch in a fresh image to understand why okToChange does not appear in the commit, but upon loading it, I get tons of `Instance of RBBlockNode did not understand #isParseError` errors that lock the image...